src/components/user/UserComponent.tsx

```diff
@@ -34,15 +35,15 @@ export default class UserComponent extends React.Component<Props, State> {
   public render() {
     const style = computeStyleSheet();
     const chipStyle = computeChipStyleSheet();
-    const { user, navigation } = this.props;
+    const { user, navigation, selected } = this.props;
     const userFullName = Utils.buildUserName(user);
     const userRole = user ? user.role : '';
     const userStatus = user ? user.status : '';
     const statusStyle = this.computeStatusStyle(userStatus, chipStyle);
     return (
-      <View style={style.container}>
+      <View style={[style.container, selected && style.selected]}>
         <View style={style.avatarContainer}>
-          <UserAvatar user={user} navigation={navigation} />
+          <UserAvatar icon={selected && 'check'} user={user} navigation={navigation} />
```

Comment:
Check if the user is selected and provide a checked user avatar.
I should have something like:
```
{selected ?
   <UserAvatar icon={'check'} user={user} navigation={navigation} />
 :
   <UserAvatar user={user} navigation={navigation} />
}
```
Or pass the check to the Avatar component

`   <UserAvatar user={user} navigation={navigation} checked={checked} />`

Reply:
I thought it was more generic this way, we could imagine other icons

Reply:
We will not change the icon once chosen.

Reply:
not necessarily for the selection, but other icons in other scenarios

Reply:
What scenarios?